src/ts/component/block/dataview/cell/text.tsx

```diff
@@ -75,14 +75,26 @@ const CellText = observer(class CellText extends React.Component<I.Cell, State>
 				EditorComponent = () => <span>{value}</span>;
 			} else 
 			if (isDate) {
-				const mask = [ '99.99.9999' ];
+				const mask = [];
 				const ph = [];
 
-				if (viewRelation.dateFormat == I.DateFormat.ShortUS) {
-					ph.push('mm.dd.yyyy');
-				} else {
-					ph.push('dd.mm.yyyy');
-				};
+				switch (viewRelation.dateFormat) {
+					case I.DateFormat.ISO: {
+						mask.push('9999.99.99');
+						ph.push('yyyy.mm.dd');
+						break;
+					}
```

Comment:
Fix formatting, add missing semicolons everywhere.

Reply:
Check PR changes carefully please, not fixed.

Reply:
![Uploading Screenshot 2024-08-13 at 15.32.28.png…]()

Reply:
Oh I'm sorry, I didn't know that `switch` statements should have a semicolon. This should be OK now.